lint/src/bin/jupiterp_linter.rs

```diff
@@ -0,0 +1,58 @@
+//! This file is part of Jupiterp: https://github.io/atcupps/Jupiterp
+
+//! This is a preliminary style checker designed for .svelte and .ts files
+//! Currently checks for ownership tag in file and 80-character limit
+
+use std::process::exit;         // exit program
+use std::env::args;             // retrieving commandline args
+use std::fs::File;              // file io
+use std::io::{prelude::*, BufReader}; 
+
+fn main() {
+    let ownership_check = false;
+    
+    if args().enumerate().count() == 1 {
+        print!("usage: jupiterp_linter <filename>\n");
+        exit(1);
+    }
+
+    // go through all arguments (names of files to read)
+    for (idx,fname) in args().enumerate() {
```

Comment:
Question: Does this require you to manually input every file you want checked? If so, that's fine for now, but the goal is to be able to input a folder like `/site` and have it check every `.ts` and `.svelte` file without having to manually write them out.